experimental/dds/tree2/src/feature-libraries/editable-tree/editableTreeBinder.ts

```diff
@@ -305,18 +326,20 @@ abstract class AbstractPathVisitor implements PathVisitor {
 	public abstract onInsert(path: UpPath, content: ProtoNodes): void;
 	public registerListener(
 		contextType: BindingContextType,
-		trees: BindTree[],
+		policies: BindPolicy[],
```

Comment:
nit: update line 319 to say `BindPolicy` rather than `BindTree`

Reply:
Done.